### DISCUSSION TITLE
API for traits validation

### DISCUSSION THREAD

**Discussion by mmeller-wikia** [ory/kratos#2170] [Ideas]
Hi all,

Currently there is no way to validate traits before updating identity.
It could be useful during the migration period to maintain data consistency between the old system and Kratos.
I would suggest implementing it as `POST / identities / validate`.

**Comment by vinckr** [comment]
Hello @mmeller-wikia 
apologies for the late answer! 
I think that would be a good addition, would you be willing to open a [Feature Request](https://github.com/ory/kratos/issues/new?assignees=&labels=feat&template=FEATURE-REQUEST.yml) with some more information around how you would like to see this feature?
